rpgJavaInterpreter-core/src/main/kotlin/com/smeup/rpgparser/parsing/parsetreetoast/misc.kt

```diff
@@ -1282,6 +1285,25 @@ internal fun CsCHECKContext.toAst(conf: ToAstConfiguration): Statement {
     )
 }
 
+private fun FactorContext.toDoubleExpression(conf: ToAstConfiguration, index: Int): Pair<Expression, Int?> =
```

Comment:
I don't understand why this function returns a pair, is it really necessary?

Reply:
I'll remove it in the next commit